scheduler/src/cook/scheduler/scheduler.clj

```diff
@@ -69,6 +69,9 @@
   ^Date []
   (tc/to-date (time/now)))
 
+; Use a separate namespace for match cycle metrics to be able to send them to a different index.
+(def match-cycle-logger-ns "cook.scheduler.scheduler-match")
```

Comment:
I changed this to use `-` as a separator so that ELK queries for `logger:cook.scheduler.scheduler` include these logs as well (using a `.` does not work since ELK does not split tokens based on that character).